include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -2948,6 +2949,48 @@ __pattern_remove_if(__parallel_tag<_IsVector> __tag, _ExecutionPolicy&& __exec,
 // merge
 //------------------------------------------------------------------------
 
+template <typename It1, typename It2, typename ItOut, typename _Comp>
+std::pair<It1, It2>
+__brick_merge_2(It1 __it_1, It1 __it_1_e, It2 __it_2, It2 __it_2_e, ItOut __it_out, ItOut __it_out_e, _Comp __comp,
```

Comment:
The naming here `__brick_merge_2`, `__pattern_merge_2`...
Can we come up with a better name to describe what is different here from the original `_XXX_merge`?
Is this only to serve the output size limited case?  if so, perhaps `__XXX_merge_size_limited`, or something similar.
I assume the the old pattern / brick is still in use?

Reply:
Or shorter and with sense about algo type. 
`__pattern_merge_path`? F.e. we already have `radix_sort` name, which is talking about algo "radix sort".